### DISCUSSION TITLE
Is there any support for dolt or could that be a feature-plugin-theme?

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [ohmyzsh/ohmyzsh#10631] [Ideas]
Hi all,

Thanks for working and maintaining ohmyzsh! I was wondering if there is any support (plugin) or theme that supports information from [dolt](https://github.com/dolthub/dolt). I was thinking about something like the theme supporting git information (like the agnoster theme) or a plugin that helps autocomplete.

Thanks.

**Comment by mcornella** [comment]
There's nothing in OMZ for dolt support. Would you like to submit a feature request?

**Reply by camilogarciabotero** [reply]
Hey @mcornella 

Thanks for your response. A FR sounds then like a good starting point!

Thanks.
